snowflake/src/main/java/org/apache/iceberg/snowflake/SnowflakeCatalog.java

```diff
@@ -48,7 +49,12 @@ public class SnowflakeCatalog extends BaseMetastoreCatalog
   // Specifies the name of a Snowflake's partner application to connect through JDBC.
   // https://docs.snowflake.com/en/user-guide/jdbc-parameters.html#application
   private static final String JDBC_APPLICATION_PROPERTY = "application";
+  // Add a suffix to user agent header for the web requests made by the jdbc driver.
+  private static final String JDBC_USER_AGENT_PROPERTY = "user_agent_suffix";
```

Comment:
nit: it's a bit non-intuitive that the property says "user agent" but the actual value says "user agent suffix"